staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/policy_decision.go

```diff
@@ -31,9 +32,11 @@ const (
 )
 
 type policyDecision struct {
-	kind    policyDecisionKind
-	message string
-	reason  metav1.StatusReason
+	kind       policyDecisionKind
+	message    string
+	reason     metav1.StatusReason
+	expression string
```

Comment:
why are we adding expression here? putting the entire cel expression in a metrics label does not seem correct

Reply:
Oh, oops, we had 'name' here before and then dropped that field so this got swapped in. Yes, we can't use this.  Let's just drop it. per-expression metrics do seem handy but we can look into that post 1.26.

Reply:
(resolving due to https://github.com/kubernetes/kubernetes/pull/113475#discussion_r1016954715 also addressing this)